src/fi/bitrite/android/ws/activity/DialogHandler.java

```diff
@@ -48,8 +48,9 @@ public Dialog createDialog(int id, String message) {
 
     private void showAlertDialog(String message) {
         AlertDialog.Builder builder = new AlertDialog.Builder(parentActivity);
-        builder.setMessage(message).setCancelable(false)
-                .setNeutralButton(parentActivity.getString(R.string.alert_neutral_button), new DialogInterface.OnClickListener() {
+        builder .setMessage(message)
+                .setCancelable(true)
+                .setPositiveButton(parentActivity.getString(R.string.alert_neutral_button), new DialogInterface.OnClickListener() {
                     public void onClick(DialogInterface dialog, int id) {
```

Comment:
Same. Single button are not centered by default now. And a single button on the left looks weird to me. (Neutral is aligned left, positive and negative to the right)